src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -451,7 +462,7 @@ static inline int MPIDI_OFI_do_put(const void *origin_addr,
         riov.key = MPIDI_OFI_winfo_mr_key(win, target_rank);
         MPIDI_OFI_INIT_CHUNK_CONTEXT(win, sigreq);
         MPIDI_OFI_CALL_RETRY(fi_writemsg(MPIDI_OFI_WIN(win).ep, &msg, flags), rdma_write, FALSE);
-        goto fn_exit;
+        goto complete_exit;
```

Comment:
I'd prefer to get rid of this extra label. We should be able to call the completion function here and then call `goto fn_exit;`, right?

Reply:
sure. i can change it

Reply:
fixed.